vispy/app/canvas.py

```diff
@@ -98,6 +99,13 @@ class Canvas(object):
         * touch
         * close
 
+    The ordering of the mouse_double_click, mouse_press, and mouse_release
+    events are not guaranteed to be consistent between backends. Only certain
+    backends natively support double-clicking (currently Qt and WX); on other
+    backends, they are detected manually with a fixed time delay.
+    This can cause problems with accessibility, as increasing the OS detection
+    time or using a dedicated double-click button will not be respected.
```

Comment:
At some point we might be able to query this info from osx/windows/linux... it looks doable for [Windows] (https://msdn.microsoft.com/en-us/library/windows/desktop/ms646258%28v=vs.85%29.aspx) and [OSX] (https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSEvent_Class/#//apple_ref/occ/clm/NSEvent/doubleClickInterval), Linux is probably harder. I'm going to assume YAGNI for now :)